templates/index.html

```diff
@@ -23,6 +23,16 @@
 					</div>
 					<div class="form-group">
 						<div class="col-xs-10 col-xs-offset-1">
+							<label for="visibility">Paste visibility: </label>
```

Comment:
I think you need to add an `{% if %}` to only show this when unlisted pastes are enabled.